### ISSUE TITLE
Web version not working?

### ISSUE THREAD

**Issue by DragonfliesProjects** [Badel2/slime_seed_finder#11]
Sorry for bothering you if I'm just stupid. 
Somehow, in Chrome and Firefox I get the same error in console and "Calculating..." without any results for hours until I just close the tab.
![image](https://user-images.githubusercontent.com/11047114/78667501-b9e52b80-78e1-11ea-93c6-13e6c81d2891.png)
Also couldn't make PC version to work because they yanked needed bitvec version probably - I keep getting error referencing it.

**Comment by Badel2** [comment]
That's possible, I only tested the web version in Firefox. The error "myWorker is not defined" suggests that your browser doesn't support web workers, or they are not enabled. My suggestion is to try it with Firefox and see if the error disappears. It should be easy to fix that error in the JavaScript code, but I don't have time right now.

> Also couldn't make PC version to work because they yanked needed bitvec version probably - I keep getting error referencing it.

This sounds bad, I'll have to investigate it, thanks reporting it.

**Comment by DragonfliesProjects** [comment]
Thank you for answer!

Same error in Firefox, I should've add it to the first post
![image](https://user-images.githubusercontent.com/11047114/78677060-f3249800-78ef-11ea-8057-b23ab6965725.png)

Also error message from my PC console:
  Installing slime_seed_finder v0.1.0 (D:\Downloads\Downloads\slime_seed_finder-master)
    Updating crates.io index
    Updating git repository `https://github.com/Badel2/anvil-region`
error: failed to compile `slime_seed_finder v0.1.0 (D:\Downloads\Downloads\slime_seed_finder-master)`, intermediate artifacts can be found at `D:\Down
loads\Downloads\slime_seed_finder-master\target`

Caused by:
  failed to select a version for the requirement `bitvec = "^0.15"`
  candidate versions found which didn't match: 0.17.4
  location searched: crates.io index
required by package `anvil-region v0.3.0 (https://github.com/Badel2/anvil-region?rev=b1edb94ca3ef8e0a74ef85eca169c995b4947e74#b1edb94c)`
    ... which is depended on by `slime_seed_finder v0.1.0 (D:\Downloads\Downloads\slime_seed_finder-master)`

Hope it helps

**Comment by Badel2** [comment]
Alright, I fixed the issues :)

The web demo had a bug introduced 10 months ago, for some reason I had never tried it. Thank you for reporting it.

The compile error was introduced 14 days ago when the author of the bitvec crate decided to delete all the versions except the last one, forcing everyone to update. So I updated and now it works.

So everything should work now. Let me know if you find any other issues!
